wazo_confd/helpers/resource.py

```diff
@@ -39,3 +39,6 @@ def delete(self, resource):
         self.validator.validate_delete(resource)
         self.dao.delete(resource)
         self.notifier.deleted(resource)
+
+    def update_form(self, form):
+        return form
```

Comment:
This layer should receive only object (sqlalchemy) or id. The form transformation is done in the restful layer